package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "BUILD_MODE=production gulp",
     "test-build": "gulp",
+    "build-its": "ts-node its/index.ts",
```

Comment:
This requires manually running these to update the IT configurations. 

It could be beneficial to run this as a CI job too to catch any changes

Reply:
I've created a [subtask](https://sonarsource.atlassian.net/browse/SONARAZDO-428) for this although I'm not certain for now how to implement it